.cirrus.yml

```diff
@@ -33,7 +33,7 @@ env:
     DEBIAN_NAME: "debian-13"
 
     # Image identifiers
-    IMAGE_SUFFIX: "c20230816t191118z-f38f37d13"
+    IMAGE_SUFFIX: "b20230821t153739z-f38f37d13"
```

Comment:
OH! This should be **c**, not b, I think? @cevich?

Reply:
oh yes, whoops I copy-pasted the wrong ID

Reply:
Opened https://github.com/containers/automation_images/pull/296